src/charts/bar.vue

```diff
@@ -29,10 +30,14 @@ export default class VueComponent extends Vue {
   private thread!: any
   private dataRows: any = {}
 
+  private plotID = this.getRandomInt(100000)
+
   private async mounted() {
     this.updateTheme()
     await this.loadData()
     this.$emit('isLoaded')
+    this.resizePlot()
+    window.addEventListener('resize', this.myEventHandler)
```

Comment:
Please add a comment line saying that we are only creating this eventListener because of the bug in plotly.

Reply:
In the docs for window.addEventListener you should have seen that the eventListener also needs to be removed when you are finished with it.

In Vue, for every eventListener that you add in mounted(), you MUST also remove it in beforeDestroy(). Otherwise the entire component never unmounts and you have a memory leak the size of Texas.

So: please add a beforeDestroy() method and include window.removeEventListener('resize', this.myEventHandler) there.